tests/functional/cnv/test_vms.py

```diff
@@ -0,0 +1,144 @@
+import logging
+import pytest
+import subprocess
+import os
+import hashlib
+from ocs_ci.framework.testlib import E2ETest
+from ocs_ci.framework.pytest_customization.marks import magenta_squad
+from ocs_ci.helpers.cnv_helpers import cal_md5sum_vm
+from ocs_ci.helpers.performance_lib import run_oc_command
+from ocs_ci.helpers.keyrotation_helper import PVKeyrotation
+from ocs_ci.utility.utils import run_cmd
+
+logger = logging.getLogger(__name__)
+
+
+class TestCNVVM(E2ETest):
+    """
+    Includes tests related to CNV workloads on MDR environment.
+
+    """
+
+    @pytest.fixture(autouse=True)
+    def setup(self, project_factory, multi_cnv_workload):
+
+        # Create a project
+        proj_obj = project_factory()
+        (
+            self.vm_objs_def,
+            self.vm_objs_aggr,
+            self.sc_obj_aggressive,
+            self.sc_obj_def_compr,
+        ) = multi_cnv_workload(namespace=proj_obj.namespace)
+
+        logger.info("All vms created successfully")
+
+    def get_md5sum(self, file_path):
+        """
+        Calculates the md5sum of the file
+        Args:
+            file_path (str): The name of the file for which md5sum to be calculated
+
+        Returns:
+            str: The md5sum of the file
+        """
+        try:
+            hash_md5 = hashlib.md5()
+            with open(file_path, "rb") as f:
+                for chunk in iter(lambda: f.read(4096), b""):
+                    hash_md5.update(chunk)
+            return hash_md5.hexdigest()
+
+        except subprocess.CalledProcessError as e:
+            print(f"Error occurred while calculating MD5 checksum: {e}")
+            return None
+
+    @magenta_squad
+    def test_cnv_vms(self, setup):
+        """
+        Tests to verify configuration for non-GS like environment
+        Steps:
```

Comment:
Please copy the steps from the approved test in the test planning [spreadsheet](https://docs.google.com/spreadsheets/d/1KX6WueORPjaOlInTLMAl1Baga64K2k_ELFGGOb7qKh0/edit?gid=882726776#gid=882726776)

Reply:
hi @hnallurv 
Here some steps(step 1-7) covered in multi-cnv-fixture. So steps which I wrote here those only I am testing in this test case.